.env.example

```diff
@@ -16,4 +16,7 @@ COMPONENTS_CLIENT_SECRET=fill-this-in
 
 # Required for Optimize API tutorial, to access API
 OPTIMIZE_BASE_URL=fill-this-in
-OPTIMIZE_AUDIENCE=https://optimize.camunda.io
\ No newline at end of file
+OPTIMIZE_AUDIENCE=https://optimize.camunda.io
+
+# Required for Web Modeler API tutorial, to access API
+MODELER_AUDIENCE=https://modeler.cloud.camunda.io/api/v1
```

Comment:
Have I mixed up audience and base URL here?

Reply:
Yes! The audience is probably `modeler.camunda.io`, but we can confirm that from the API docs.